includes/definitions/discovery/baicells-od04.yaml

```diff
@@ -0,0 +1,127 @@
+mib: BAICELLS-CPE-MIB
+modules:
+    os:
+        hardware: BAICELLS-CPE-MIB::HardwareVersion.0
+        serial: BAICELLS-CPE-MIB::SerialNumber.0
+        version: BAICELLS-CPE-MIB::SoftwareVersion.0
+    sensors:
+        runtime:
+            data:
+                -
+                    oid: UpTime
+                    num_oid: .1.3.6.1.4.1.53058.2.1.9.0
```

Comment:
What is the point of this?  There is already SNMPv2-MIB::sysUpTime.0.  If for some reason the device doesn't supply that, you can substitute this oid.

Reply:
Sure, this can be removed, I had it here mostly for testing purposes. The device is Linux-based and supplies the OID you mentioned.